chapter_optimization/lr-scheduler_vn.md

```diff
@@ -383,15 +383,19 @@ Unfortunately this means that progress is slow.
 Conversely, a large learning rate initially leads to divergence.
 -->
 
-*dịch đoạn phía trên*
+Trong một số trường hợp, khởi tạo tham số không đảm bảo sẽ có kết quả tốt. 
+Đặc biệt đối với các mạng phức tạp, nó có thể làm việc tối ưu hóa không ổn định. 
+Chúng ta có thể giải quyết việc này bằng cách chọn tốc độ học đủ nhỏ để ngăn phân kỳ vào lúc bắt đầu. Tuy nhiên, tiến trình học sẽ chậm hơn. 
+Ngược lại, tốc độ học lớn ban đầu cũng gây ra phân kỳ.```
 
 <!--
 A rather simple fix for this dilemma is to use a warmup period during which the learning rate *increases* to its initial maximum and to cool down the rate until the end of the optimization process.
 For simplicity one typically uses a linear increase for this purpose.
 This leads to a schedule of the form indicated below.
 -->
-
-*dịch đoạn phía trên*
+Một giải pháp đơn giản cho vấn đề trên là dùng quá trình khởi động (warmup) vào khoảng thời gian tốc độ học tăng tới giá trị lớn nhất rồi giảm dần tới khi kết thúc quá trình tối ưu.
```

Comment:
```suggestion
Một giải pháp đơn giản cho vấn đề trên là dùng quá trình khởi động (*warmup*), trong thời gian đó tốc độ học *tăng* tới giá trị lớn nhất, sau đó giảm dần tới khi kết thúc quá trình tối ưu.
```
bạn dịch sai ý `during which` ở câu gốc:
>A rather simple fix for this dilemma is to use a warmup period **during which** the learning rate *increases* to its initial maximum and to cool down the rate until the end of the optimization process.

mình nghĩ câu gốc có vấn đề về ngữ pháp ở phần cuối `to cool down the rate until the end of the optimization process`. Đọc code phía dưới và nhìn hình kết quả thì mình đề xuất dịch như trên. Còn chú thích từ gốc tiếng Anh nhóm thống nhất in nghiêng nên bạn lưu ý format: `*warmup*`

![image](https://user-images.githubusercontent.com/30251584/86798606-29161b80-c09b-11ea-9b94-61d63d69230d.png)